An error has occurred:
The machine terminated because of an error, either from a built-in function or from an explicit use of 'error'.
Caused by: error

Trace:
-> func (test/CallTrace/Spec.hs:79:1-79:4)
-> $cmyClassFunc (test/CallTrace/Spec.hs:100:3-100:13)
-> error